notebooks/00_quick_start/wide_deep_movielens.ipynb

```diff
@@ -0,0 +1,1225 @@
+{
```

Comment:
@loomlike we are hosting all the images in a blob to reduce the size of the wiki. Ideally in svg. See an example: https://github.com/Microsoft/Recommenders/blob/master/notebooks/02_model/ncf_deep_dive.ipynb

The location and key is on teams, please let me know if you need help on this.

Reply:
Updated!